rm/rm-server/src/test/java/functionaltests/nodesource/TestSSHInfrastructureV2.java

```diff
@@ -179,7 +179,7 @@ public void testSSHInfrastructureV2WithRestartDownNodes() throws Exception {
         RMTHelper.log("Final checks on the scheduler state");
         s = resourceManager.getState();
         assertEquals(NB_NODES, s.getTotalNodesNumber());
-        assertEquals(NB_NODES, s.getFreeNodesNumber());
+        assertEquals(NB_NODES, s.getTotalAliveNodesNumber()); // check amount of all nodes that are not down
```

Comment:
In fact we can not from resourceManager.getState() retrieve: pending, busy nodes amount.
But according to method RMCore::listAliveNodeUrls() it returns all nodes that are not down. I suppose that we expect to check exactly this behavior in this test - the amount of nodes that are not down.